src/__tests__/Quote.test.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+import Quote from '../pages/Quote';
+
+describe('Testing the Quote page:', () => {
+  test('should renerd correctly', () => {
+    render(<Quote />);
+    expect(screen).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job adding tests for Quote, Home etc. But you need to cover all the functionalities & scenarios of the operate.js and calculate.js modules

Reply:
Thank you @GzimAsani , I'll definitely work on that 💪